Examples/Algorithms/EventRecording/src/PrimaryGeneratorAction.hpp

```diff
@@ -0,0 +1,55 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <memory>
+#include "G4ParticleTable.hh"
+#include "G4SystemOfUnits.hh"
+#include "G4ThreeVector.hh"
+#include "G4VUserPrimaryGeneratorAction.hh"
+#include "globals.hh"
+
+class G4ParticleGun;
+class G4Event;
+
+namespace ActsExamples {
+
+/// @class PrimaryGeneratorAction
+///
```

Comment:
Unnecessary. The comment is automatically assigned to the correct class.

Reply:
done